data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/certificate/CertificateProviderFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  The OpenSearch Contributors require contributions made to
+ *  this file be licensed under the Apache-2.0 license or a
+ *  compatible open source license.
+ *
+ *  Modifications Copyright OpenSearch Contributors. See
+ *  GitHub history for details.
+ */
+
+package com.amazon.dataprepper.plugins.source.loghttp.certificate;
```

Comment:
Not a blocker for now, but do we want to centralize the SSL certificate logic so we don't have these CertificateProviderFactories all over the codebase?